evaluation/eval.py

```diff
@@ -38,7 +38,11 @@ class EvaluationArguments:
     tag: Optional[str] = field(
         default=None,
         metadata={"help": "Identifier for the evaluation run."}
-    )    
+    )
+    is_english_only: Optional[bool] = field(
```

Comment:
I think `english_only` would suffice here without the extra "is" in the front

Reply:
Agree that it's a bit verbose. Fixed now!